### ISSUE TITLE
loc_height vs. loc_altitude

### ISSUE THREAD

**Issue by rbair23** [mlagally/W3C-Work#7]
"height" in ISO 6709 is pretty loose in its interpretation, while "altitude" is specific in the same way that "latitude" and "longitude" are specific. I think "altitude" is a better fit here.

In addition, we shouldn't require all three values if any one value is available. Many devices can give lat/lon but not altitude, or vice versa. If we're going to define loc_altitude, we should define loc_height and loc_depth as well, and make those three values optional.

**Comment by mlagally** [comment]
I agree, we should include altitude and depth and make all 3 optional.

However I would discourage the use of altitude, since it talks about mean sea level which is not a worldwide identical reference point.
See:
https://www.science20.com/news_articles/what_happens_bridge_when_one_side_uses_mediterranean_sea_level_and_another_north_sea-121600
